pandas/tests/frame/test_arithmetic.py

```diff
@@ -211,6 +213,17 @@ def test_df_sub_datetime64_not_ns(self):
                                  pd.Timedelta(days=2)])
         tm.assert_frame_equal(res, expected)
 
+    @pytest.mark.parametrize('other', [
+        pd.Timedelta('1d'),
+        datetime.timedelta(days=1),
+        np.timedelta64(1, 'D')
+    ])
+    def test_timestamp_df_add_timedelta(self, other):
+        # GH 22005
+        expected = pd.DataFrame([pd.Timestamp('2018-01-02')])
+        result = pd.DataFrame([pd.Timestamp('2018-01-01')]) + other
```

Comment:
can you add the reverse test as well (other + DataFrame...)

Reply:
While you're at it, might as well do subtraction too